index.py

```diff
@@ -24,6 +24,7 @@
 ROOT = Path(__file__).resolve().parent
 GIT_PATH = getenv('GIT_PATH', which('git'))
 SKIP_FILES = {'index.md', 'README.md', 'CONTRIBUTING.md'}
+SKIP_DIRECTORY = Path('_site').resolve()
```

Comment:
```suggestion
SKIP_DIRECTORY = ROOT / '_site'
```

Sinon ça ne fonctionne plus si on lance le programme sans être dans le même dossier que lui, c'est justement à ça que `ROOT` sert

Reply:
Mais en fait, c'est quoi ce dossier `_site` ? Il y a des fichiers markdown dedans ?!

Reply:
Le dossier _site c'est le dossier dans lequel les fichiers html générés par jekyll se trouvent. Cependant je trouve également les fichiers markdowns

Reply:
Y a peut-être moyen d'empêcher jekyll de faire une copie des markdowns dans ce dossier j'avais pas regardé à ce moment là

Reply:
En fait on dirait une sorte d'optimisation pour ne pas indexer les fichiers qui se trouve dans le dossier `_site`. Ces changements auraient du être discuté d'abord dans une issue et être faits dans un commit à part. D'ailleurs je ne suis même pas sur qu'on en a besoin parce que je ne pense pas qu'il y a de fichiers json dans ce dossier.

Reply:
edit: j'ai pas du la réponse, ok donc il y a bel et bien un problème pour ceux qui lancent ce script en ayant jekyll sur leur machine. c'est un bug qui doit être discuté à part de cette PR

Reply:
Quel problème les markdowns dans _site? Sinon oui j'ai modifié discover_articles pour pas indexer les markdowns qui sont dans _site

Reply:
> Sinon oui j'ai modifié discover_articles pour pas indexer les markdowns qui sont dans _site

Le problème c'est que ces modifications ne sont documentées nul le part, on doit le deviner de notre cul pourquoi tu as fait ces modifications

Reply:
et aussi que le code est dégueulasse

Reply:
Ah j'aurais peut-être dû mettre des commentaires c'est vrai

Reply:
> et aussi que le code est dégueulasse

C'est vrai que j'ai plus essayé de faire un truc fonctionnel que propre

Reply:
> Ah j'aurais peut-être dû mettre des commentaires c'est vrai

Non, c'est changements auraient du être dans un commit à part et correctement documentés dans le message de commit

Reply:
https://github.com/Julien00859/teaching-git/wiki/explain-commit-message